From cc49d6ef00c35541e198abacdedf0c95f0d0f957 Mon Sep 17 00:00:00 2001
From: Jeffery <jeffery_wu@tw.shuttle.com>
Date: Wed, 12 Oct 2016 10:25:18 +0800
Subject: [PATCH 313/443] Fix Bug: After adding a new Role by 'Save and Add
 another', system should save the editted role/sort and clean the all columns
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

    Desciption:
    如果選擇繼續建立,則新增新角色後畫面停留在本畫面,但清空角色名稱&排序欄位
---
 templates/role_add.html | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/templates/role_add.html b/templates/role_add.html
index 9f80f77..b6ed086 100644
--- a/templates/role_add.html
+++ b/templates/role_add.html
@@ -43,7 +43,7 @@
                                             <i style="color: red;">*</i> {% trans "Sort" %}
                                         </label>
                                         <div class="col-md-6">
-                                            <input type="number" class="form-control" placeholder="{% trans "Please input the sort number." %}" title="{% trans "Please input the sort number." %}" name="Sort" style="ime-mode:disabled" min="1" max="100" oninput="setCustomValidity('')" oninvalid="setCustomValidity(`{% trans "The sort number can only be positive integer." %}`)" required>
+                                            <input type="number" class="form-control" value="{}" placeholder="{% trans "Please input the sort number." %}" title="{% trans "Please input the sort number." %}" name="Sort" style="ime-mode:disabled" min="1" max="100" oninput="setCustomValidity('')" oninvalid="setCustomValidity(`{% trans "The sort number can only be positive integer." %}`)" required>
                                         </div>
                                     </div>
                                     <div class="col-md-offset-3 col-md-9">
@@ -98,7 +98,7 @@
                     if (data.CODE == "0") {
                         alert("{% trans "Role add successfully" %}");
                         if (addAnother) {
-                            location.reload();
+                            location.replace(location);
                         } else {
                             location.href = '/role_list/';
                         }
-- 
1.9.1

